multimodal/src/autogluon/multimodal/utils/log.py

```diff
@@ -200,10 +200,13 @@ def get_gpu_message(detected_num_gpus: int, used_num_gpus: int):
     A string with the GPU info.
     """
     gpu_message = f"{detected_num_gpus} GPUs are detected, and {used_num_gpus} GPUs will be used.\n"
-    for i in range(detected_num_gpus):
-        free_memory, total_memory = torch.cuda.mem_get_info(i)
-        gpu_message += f"   - GPU {i} name: {torch.cuda.get_device_name(i)}\n"
-        gpu_message += f"   - GPU {i} memory: {free_memory * 1e-9:.2f}GB/{total_memory * 1e-9:.2f}GB (Free/Total)\n"
+    if not any(s in strategy for s in ["fork", "notebook"]):
```

Comment:
Do we not want any logs in notebook?

Reply:
Calling `torch.cuda.mem_get_info()` initializes CUDA, which would trigger one pytorch error "Cannot re-initialize CUDA in forked subprocess".

Reply:
See https://github.com/Lightning-AI/lightning/blob/6b7abda5a3be5ee3204c8e1ca5a8cd644e768ba4/src/lightning/fabric/strategies/launchers/multiprocessing.py#L186-L203

Reply:
I see.